### ISSUE TITLE
raison d'etre should have a circumflex

### ISSUE THREAD

**Issue by hugobuddel** [astronomyk/ScopeSim-SPIE-paper#2]
https://github.com/astronomyk/ScopeSim-SPIE-paper/blob/95c1216f14ae3810f0fab22f37238f35e81be28e/0_abstract.tex#L54

**Comment by astronomyk** [comment]
Thanks. Will change it in the latex version, but for a text field on a web form, this won't matter

**Comment by hugobuddel** [comment]
Yeah didn't realize you're actually writing markdown or rst and not latex :-).

I personally try to refrain from using foreign languages in my English, because I never know whether using a phrase like raison d'être is just a Dutch thing or also an English thing.
